src/scss/aruba/_aruba.defaults.scss

```diff
@@ -19,7 +19,7 @@ $brand-status-colors: (
   disabled: #999999
 );
 $brand-link-color: $brand-color;
-$brand-grey-colors: (#2d3034, #52555a, #999999);
+$brand-grey-colors: (#2d3034, #52555a, #999999, #666666);
```

Comment:
I am using the 4th brand grey color of the HPE default. When I run gulp dist though, the build process is throwing an error because aruba does not have a 4th grey, even though it is not being used. I can hardcode the grey color if that is preferential